source/Energinet.DataHub.MarketParticipant.Application/Handlers/Actor/UpdateActorHandler.cs

```diff
@@ -81,6 +84,8 @@ await _externalActorIdConfigurationService
                 .AssignExternalActorIdAsync(actor)
                 .ConfigureAwait(false);
 
+            await _uniqeMarketRoleGridAreaService.EnsureUniqueMarketRolesPerGridAreaAsync(actor).ConfigureAwait(false);
```

Comment:
Fix spelling mistake

```suggestion
            await _uniqueMarketRoleGridAreaService.EnsureUniqueMarketRolesPerGridAreaAsync(actor).ConfigureAwait(false);
```